tests/test_bmi_visualization_guidance_branches.py

```
@@ -50,9 +52,10 @@ def test_guidance_chart_covers_all_recommendation_branches():
             lang="ru",
         )
-    finally:
         plt.close(fig_mid)
 
-    assert ax_low.texts, "expected annotations for low BMI scenario"
-    assert ax_high.texts, "expected annotations for high BMI scenario"
-    assert ax_mid.texts, "expected annotations for mid BMI scenario"
+        assert ax_low.texts, "expected annotations for low BMI scenario"
+        assert ax_high.texts, "expected annotations for high BMI scenario"
+        assert ax_mid.texts, "expected annotations for mid BMI scenario"
+    except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)